src/particles/diagnostics/BeamRelevant.cpp

```diff
@@ -0,0 +1,225 @@
+
+                        const ParticleReal p_xux = (p_x-x_mean)*(p_ux-ux_mean)*p_w;
+                        const ParticleReal p_yuy = (p_y-y_mean)*(p_uy-uy_mean)*p_w;
+                        const ParticleReal p_zuz = (p_t-t_mean)*(p_ut-ut_mean)*p_w;
+
+                        const ParticleReal p_charge = q_qe*p_w;
+
+                        return {p_x_ms, p_y_ms, p_z_ms,
+                                p_ux_ms, p_uy_ms, p_uz_ms,
+                                p_gm_ms,
+                                p_xux, p_yuy, p_zuz,
+                                p_charge};
+                    },
+                    reduce_ops2);
+
+            std::vector<ParticleReal> values_per_rank_2nd = {
+                    amrex::get<0>(r2), // x_ms
+                    amrex::get<1>(r2), // y_ms
+                    amrex::get<2>(r2), // z_ms
+                    amrex::get<3>(r2), // ux_ms
+                    amrex::get<4>(r2), // uy_ms
+                    amrex::get<5>(r2), // uz_ms
+                    amrex::get<6>(r2), // gm_ms
+                    amrex::get<7>(r2), // xux
+                    amrex::get<8>(r2), // yuy
+                    amrex::get<9>(r2), // zuz
+                    amrex::get<10>(r2) // charge
+            };
+
+            // reduced sum over mpi ranks (reduce to IO rank)
+            ParallelDescriptor::ReduceRealSum
+                    ( values_per_rank_2nd.data(), values_per_rank_2nd.size(), ParallelDescriptor::IOProcessorNumber());
+
+            ParticleReal x_ms   = values_per_rank_2nd.at(0) /= w_sum;
+            ParticleReal y_ms   = values_per_rank_2nd.at(1) /= w_sum;
+            ParticleReal t_ms   = values_per_rank_2nd.at(2) /= w_sum;
+            ParticleReal ux_ms  = values_per_rank_2nd.at(3) /= w_sum;
+            ParticleReal uy_ms  = values_per_rank_2nd.at(4) /= w_sum;
+            ParticleReal ut_ms  = values_per_rank_2nd.at(5) /= w_sum;
+            ParticleReal gm_ms  = values_per_rank_2nd.at(6) /= w_sum;
+            ParticleReal xux    = values_per_rank_2nd.at(7) /= w_sum;
+            ParticleReal yuy    = values_per_rank_2nd.at(8) /= w_sum;
+            ParticleReal tut    = values_per_rank_2nd.at(9) /= w_sum;
+            ParticleReal charge = values_per_rank_2nd.at(10);
+            // standard deviations of positions
+            ParticleReal sig_x = std::sqrt(x_ms);
+            ParticleReal sig_y = std::sqrt(y_ms);
+            ParticleReal sig_t = std::sqrt(t_ms);
+            // standard deviations of momenta
+            ParticleReal sig_px = std::sqrt(ux_ms);
+            ParticleReal sig_py = std::sqrt(uy_ms);
+            ParticleReal sig_pt = std::sqrt(ut_ms);
+            // RMS emittances
+            ParticleReal emittance_x = std::sqrt(x_ms*ux_ms-xux*xux);
+            ParticleReal emittance_y = std::sqrt(y_ms*uy_ms-yuy*yuy);
+            ParticleReal emittance_z = std::sqrt(t_ms*ut_ms-tut*tut);
+
```

Comment:
Thanks, @cemitch99! I am printing out beta*gamma now as well so that we can just compute this offline, if that's okay. :)
This pull request is going to be an intermediate step towards the output in openPMD but if anyone is going to use it during this stage, we have it available now as well. 👍